internal/hcl/graph_vertex_module_call.go

```diff
@@ -78,8 +78,6 @@ func (v *VertexModuleCall) Visit(mutex *sync.Mutex) error {
 }
 
 func (v *VertexModuleCall) evaluate(e *Evaluator, b *Block, mutex *sync.Mutex) error {
-	mutex.Lock()
```

Comment:
Is the removal of these locks now because the visits are not concurrent? Or is there another lock I'm missing somewhere.

Reply:
Yeah exactly, they're sequential at the moment. We could parallelise the walk of the graph in future, but I didn't go that far for now.